html/browser/delay_interactivity.js

```diff
@@ -0,0 +1,24 @@
+// Disables all links for one second after the browser window opens.
+
+(function(){
+  // If there's already an onload, let's not clobber it
+  var oldonload = window.onload
+  window.onload = function(){
+    if(typeof oldonload == 'function')
+      oldonload();
```

Comment:
@Krausus if you're going to do single-statement, put it on the right of the if
don't do this horrific tabbing pls ;W;